app/services/clients/products_client.js

```diff
@@ -43,11 +45,13 @@ module.exports = {
  */
 function createProduct (options) {
   return baseClient.post({
+    headers,
     baseUrl,
     url: `/products`,
     json: true,
     body: {
       gateway_account_id: options.gatewayAccountId,
+      pay_api_token: options.payApiToken,
```

Comment:
I think it was the lack of the `pay_api_token` that was making it break rather than the lack of headers.

Reply:
This is  a different token (publicapi bearer token for creating a payment). This is still needed. we removed the need for having `PRODUCTS_API_TOKEN` which was in header an no longer exist.

Reply:
oh i see what you mean @maxcbc . Why on earth was this removed??